db/redis.js

```diff
@@ -10,21 +10,21 @@ promisifyAll(redis.Multi.prototype);
 const client = redis.createClient(config.client);
 
 client.on('ready', () => {
-	loggerRedis.info('Redis is ready');
+	if (loggerRedis) loggerRedis.info('Redis is ready');
 });
 
 client.on('connect', () => {
-	loggerRedis.verbose('Connect to redis');
+	if (loggerRedis) loggerRedis.verbose('Connect to redis');
 	if (config.client.password) {
 		client.auth(config.client.password, () => {
-			loggerRedis.verbose('Authenticated to redis');
+			if (loggerRedis) loggerRedis.verbose('Authenticated to redis');
```

Comment:
Why did you add this? because redis gets initialized before logger?

Reply:
Yeah after importing redis into `constants.js`, errors were being thrown because of the logger

Reply:
What are the errors. Its important to see if redis is connected for log checking. This way it might just skip those logs.

Reply:
Actually I only needed to add this check to `pubsub.js` and not `redis.js`. The error message was:

`hollaex-server_1    | TypeError: Cannot read property 'verbose' of undefined
hollaex-server_1    |     at RedisClient.client.on (/app/db/pubsub.js:12:14)
hollaex-server_1    |     at RedisClient.emit (events.js:189:13)
hollaex-server_1    |     at RedisClient.on_connect (/app/node_modules/redis/index.js:422:10)
hollaex-server_1    |     at Socket.<anonymous> (/app/node_modules/redis/index.js:268:14)
hollaex-server_1    |     at Object.onceWrapper (events.js:277:13)
hollaex-server_1    |     at Socket.emit (events.js:194:15)
hollaex-server_1    |     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1083:10)
hollaex-db_1        | 2020-03-06 07:35:02.472 UTC [39] LOG:  incomplete startup packet
hollaex-server_1    | 2020-03-06T07:35:02: PM2 log: App name:api id:0 disconnected
hollaex-server_1    | 2020-03-06T07:35:02: PM2 log: App [api:0] exited with code [0] via signal [SIGINT]
hollaex-server_1    | 2020-03-06T07:35:02: PM2 log: App [api:0] starting in -cluster mode-`

It was making the server restart over and over again.

Reply:
So we can remove this now from redis?

Reply:
@abeikverdi Yeah i'll make a quick commit